sycl/include/sycl/detail/backend_traits_level_zero.hpp

```diff
@@ -207,6 +208,12 @@ template <> struct BackendReturn<backend::ext_oneapi_level_zero, kernel> {
   using type = ze_kernel_handle_t;
 };
 
+using graph = ext::oneapi::experimental::command_graph<
+    ext::oneapi::experimental::graph_state::executable>;
+template <> struct BackendReturn<backend::ext_oneapi_level_zero, graph> {
+  using type = ze_command_list_handle_t;
```

Comment:
Should this be documented in the [L0 backend specification](https://github.com/intel/llvm/blob/sycl/sycl/doc/extensions/supported/sycl_ext_oneapi_backend_level_zero.md#4-interoperability-with-the-level-zero-api)?

Reply:
Updated that doc now